### PR TITLE
feature: revert temporary Cyprus fork code

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: plugins/coresystem/src/validators/NemesisSinkValidator.cpp [hunk 1]
```diff
@@ -36,7 +36,7 @@ namespace catapult { namespace validators {
 			if (!isNemesisPublicKey || Height(1) == context.Height)
 				return ValidationResult::Success;
 
-			if (additionalAllowedSignaturesHeight != context.Height && Height() != context.Height)
+			if (additionalAllowedSignaturesHeight != context.Height)
```
Comment (gimre-xymcity) [inline plugins/coresystem/src/validators/NemesisSinkValidator.cpp:39]: is this one correct?

File: src/catapult/model/NotificationPublisher.cpp [hunk 2]
```diff
@@ -35,15 +35,7 @@ namespace catapult { namespace model {
...
 
 		BlockNotification CreateBlockNotification(const Block& block, const Address& blockSignerAddress) {
-			return {
-				block.Type,
-				blockSignerAddress,
-				block.BeneficiaryAddress,
-				block.Timestamp,
-				block.Difficulty,
-				block.FeeMultiplier,
-				block.TransactionsHash
```
Comment (gimre-xymcity) [inline src/catapult/model/NotificationPublisher.cpp:45]: wondering if we should leave this notification as is

File: plugins/coresystem/src/validators/NemesisSinkValidator.cpp (hunk 1 above)
Comment (Jaguar0625) [reply plugins/coresystem/src/validators/NemesisSinkValidator.cpp:39]: yea, 0-height check was only for UTs, which are now passed

File: src/catapult/model/NotificationPublisher.cpp (hunk 2 above)
Comment (Jaguar0625) [reply src/catapult/model/NotificationPublisher.cpp:45]: we could always add it back again if needed

**Review by gimre-xymcity — APPROVED** [review]

**Pull request merged**